Bearded.Monads/TryExtensions.cs

```diff
@@ -192,6 +192,10 @@ public static A ElseThrow<A>(this Try<A> either)
             this Try<Try<A>> ee)
             => ee.SelectMany(id);
 
+        public static Try<IEnumerable<A>> Flatten<A>(
+            this Try<IEnumerable<IEnumerable<A>>> incoming)
+            => incoming.Select(x => x.SelectMany(id));
```

Comment:
Why not just flatten?

![like this](https://media.giphy.com/media/12JxbhSrnaWf8A/giphy.gif)

Reply:
I can't find a way of doing it with any other already defined Flatten in the library.

Reply:
Oh! Silly me - would you mind creating an `EnumerableExtensions.cs` file and defining the `Flatten` for `IEnumerable<IEnumerable<A>>` in there?

Reply:
The reason I ask is that if there's a specific implementation provided for `Try` that implementation will bleed out into other implementations for other types - better to have it in one spot, no?

Reply:
I agree :) 
Changes done and pushed.